docs/README.md

```diff
@@ -0,0 +1,58 @@
+Docco Documentation
+-----------------------------
+
+Docco is listed in `devDependences` in `package.json` and should install as part of `npm install`. Docco converts comments into inline documentation in static HTML pages located in the `/docs` folder of this repo. Documentation has been checked in to the repo to provide easy timestamping for the last build of the docs.
+
```

Comment:
I don't understand what "Documentation has been checked in to the repo to provide easy timestamping for the last build of the docs" means exactly -- I'm not sure I like generated docs checked in to the `master` branch